submarine-client/src/main/java/org/apache/submarine/client/cli/runjob/RunJobCli.java

```diff
@@ -219,6 +224,23 @@ private void addTensorboardOptions(Options options) {
             CAN_BE_USED_WITH_TF_ONLY);
   }
 
+  private void addSchedulerOptions(Options options) {
+    options.addOption(CliConstants.N_SCHEDULERS, true,
+        "Number of scheduler tasks of the job. " +
+        "It should be 1 or 0, by default it's 0."+
+        CAN_BE_USED_WITH_MXNET_ONLY);
+    options.addOption(CliConstants.SCHEDULER_DOCKER_IMAGE, true,
+        "Specify docker image for scheduler, when this is not specified, " +
+        "scheduler uses --" + CliConstants.DOCKER_IMAGE +
+        " as default. " + CAN_BE_USED_WITH_MXNET_ONLY);
+    options.addOption(CliConstants.SCHEDULER_LAUNCH_CMD, true,
+        "Commandline of scheduler, arguments will be " +
+        "directly used to launch the scheduler. " + CAN_BE_USED_WITH_MXNET_ONLY);
+    options.addOption(CliConstants.SCHEDULER_RES, true,
+        "Resource of each scheduler, for example " +
+        "memory-mb=2048,vcores=2,yarn.io/gpu=2. " + CAN_BE_USED_WITH_MXNET_ONLY);
```

Comment:
Will MXnet scheduler server use GPU? if not, we can remove `yarn.io/gpu=2`

Reply:
Thanks for the review! I will remove it.